prime-router/metadata/HL7/catchall/hl7/segments/OBX/OBXExtension.yml

```diff
@@ -165,7 +165,46 @@ extension:
           type: STRING
           expressionType: HL7Spec
           valueOf: OBX.29
-
+    - expressionType: nested
+      vars:
+        obx31: STRING, OBX.31
+      condition: $obx31 NOT_NULL
+      expressionsMap:
+        url:
+          type: STRING
+          value: OBX.31
+        valueId:
+          type: STRING
+          expressionType: HL7Spec
+          valueOf: OBX.31
+    - expressionType: nested
+      vars:
+        obx32: STRING_ALL, OBX.32
+      condition: $obx32 NOT_NULL
+      generateList: true
+      specs: OBX.32 *
+      expressionsMap:
+        url:
+          type: STRING
+          value: OBX.32
+        valueCodeableConcept:
+          valueOf: datatypes/CWE/CodeableConcept
+          expressionType: resource
+    - expressionType: nested
```

Comment:
Deferring work to implement the inventory mapping for OBX-33 for a later issue. In the meantime, this extension preserves the field for HL7 -> FHIR -> HL7 conversion.

Reply:
Can you please link the ticket?

Reply:
Created #14569